library/java/src/io/envoyproxy/envoymobile/engine/JvmStringAccessorContext.java

```diff
@@ -0,0 +1,18 @@
+package io.envoyproxy.envoymobile.engine;
+
+import java.nio.ByteBuffer;
+
+import io.envoyproxy.envoymobile.engine.types.EnvoyStringAccessor;
+
+class JvmStringAccessorContext {
```

Comment:
Given that this accessor runs on the threading context that calls it I am not sure if we need it versus just passing the EnvoyStringAccessor to the jni and retaining that.

Reply:
The concrete class allows methods to be cached.